### ISSUE TITLE
Question: file input and output?

### ISSUE THREAD

**Issue by buzzdev** [facebookresearch/denoiser#122]
Is there any way to let this read the input from an audio file and write the output to another file?
Thank you

**Comment by qalabeabbas49** [comment]
yes, you can use the following code. It will enhance/denoise all the files in noisy_dir and output the clean files in out_dir.

`python -m denoiser.enhance --model_path=<path to the model> --noisy_dir=<path to the dir with the noisy files> --out_dir=<path to store enhanced files>
`

**Comment by nguyenvulong** [comment]
I was able to run it without `--model_path`. It worked well. Thank you. This should be on the `README.md`.


it seems like the pretrained model was loaded somewhere .. if anyone was aware of it please let me know.

```
❯ python -m denoiser.enhance --noisy_dir=./noisy --out_dir=./nonoisy
INFO:denoiser.pretrained:Loading pre-trained real time H=48 model trained on DNS.
INFO:__main__:Generate enhanced files | 1/1 | 50.8 it/sec
Waiting for pending jobs...
INFO:__main__:Generate enhanced files | 1/1 | 1.7 it/sec
```
